core/rawdb/freezer.go

```diff
@@ -597,5 +597,5 @@ func gcKvStore(db ethdb.KeyValueStore, ancients []common.Hash, first uint64, fro
 	if n := len(ancients); n > 0 {
 		context = append(context, []interface{}{"hash", ancients[n-1]}...)
 	}
-	log.Info("Deep froze chain segment", context...)
+	log.Debug("Deep froze chain segment", context...)
```

Comment:
This log would be printed periodically, in case of deadlock, it could be helpful to check if the process is still alive.
And there won't be too much of this log, right? so maybe we can keep it.

Reply:
Yup, it doesn't appear much, changed back to "info"